README.md

```diff
@@ -69,6 +69,23 @@ following command:
 npm run test:jest -- -u
 ```
 
+## Deployment
+
+The production version of the Glean Dictionary
+(https://dictionary.telemetry.mozilla.org) is deployed from the `production`
+branch on this repository, which usually corresponds to the latest GitHub
+release. To update the Glean Dictionary to the latest version, follow this
+procedure:
+
+- Create a new release (you might find
+  [this iodide notebook](https://alpha.iodide.io/notebooks/6616/) useful for
+  creating a set of notes on what changed)
+- From a local checkout, update the `production` branch to be in sync with the
+  tag you just created, then push to the production branch.
```

Comment:
is the production branch automatically deployed to dictionary.telemetry.mozilla.org or do I need to ask anyone on SRE to deploy it?

Reply:
It will be deployed automatically, added some clarification.